transport/http/outbound.go

```diff
@@ -230,3 +236,48 @@ func (o *outbound) Call(ctx context.Context, treq *transport.Request) (*transpor
 
 	return nil, errors.RemoteUnexpectedError(message)
 }
+
+type ack struct{}
+
+func (ack) String() string {
+	return ""
```

Comment:
I would put a non-empty string here. Maybe the time at which the request was
accepted by the server?

```
type ack struct{t time.Time}
```